pkg/cluster/networkprofile.go

```diff
@@ -0,0 +1,57 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/util/feature"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// PopulateMTUSize ensures that every new cluster object has the MTUSize field defined
+func (m *manager) populateMTUSize(ctx context.Context) error {
+	// Get appropriate MTU size
+	mtuSize := api.MTU1500
+	subProperties := m.subscriptionDoc.Subscription.Properties
+	if feature.IsRegisteredForFeature(subProperties, api.FeatureFlagMTU3900) {
+		mtuSize = api.MTU3900
+	}
+
+	// Patch the cluster object with correct MTU size
+	_, err := m.db.PatchWithLease(ctx, m.doc.Key, func(doc *api.OpenShiftClusterDocument) error {
+		doc.OpenShiftCluster.Properties.NetworkProfile.MTUSize = mtuSize
+		return nil
+	})
+	return err
+}
+
+// EnsureMTUSize ensures that an existing cluster object has the MTUSize field defined
+func (m *manager) EnsureMTUSize(ctx context.Context) error {
```

Comment:
```suggestion
func (m *manager) ensureMTUSize(ctx context.Context) error {
```

Should be able to use lowercase as it's in the same package as where you're calling it.